app/models/pano/email_validator.rb

```diff
@@ -1,6 +1,6 @@
 module Pano
   class EmailValidator < ActiveModel::EachValidator
-    EMAIL_REGEX = /\A[a-z0-9][a-z0-9._%+-]*(?<!\.)@(?:[-a-z0-9]+\.)+[a-z]{2,}\z/i
+    EMAIL_REGEX = /\A([\w+\-].?)+(?<!\.)@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*\z/i
 
```

Comment:
This regex does not seem to be correct. For example, I believe it would pass the following
`dave isaacs@gmail.com`
`dave@isaacs@gmail.com`
`there's no way this is an email @example`

I think the big culprit is the  `.?` in the first part (`\A([\w+\-].?)+ ...`), which allows any character any number of times. But there may be other problems beyond that, because I also managed to pass an invalid domain name.